.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       if: matrix.os == 'windows-latest'
 
     - name: build
-      run: cmake --build out --config Release
+      run: cmake --build out --config Release -v
```

Comment:
Debugging?

Reply:
Yes, but I actually prefer having automated build logs be as verbose as possible. It doesn't really cost anything and anytime you have to debug any issue, usually the first thing you have to do is reproduce it with verbose to see the flags.

Reply:
What does `-v` do when running cmake exactly?    We should probably use this flag everywhere or nowhere.